docs/bridge/relayer_service.mdx

```diff
@@ -3,16 +3,22 @@ title: Relayer Service
 sidebar_position: 2
 ---
 
-The relayer is probably the simplest component of the bridge. It receives blocks from a source chain and sends those blocks to a light client contract on the destination chain, establishing a bidirectional bridge between the chains. In order to accomplish this, light client contracts must exist on both chains.
+The relayer is a fundamental component of the bridge, responsible for establishing bidirectional communication between chains. It receives blocks from a source chain and relays them to a light client contract on the destination chain. 
+This enables seamless interoperability and data transfer between the two chains. It is important to note that light client contracts must be deployed on both the source and destination chains for the relayer to function effectively.
 
-The NEAR to Calimero relayer reads latest blocks on the NEAR blockchain and sends (relays) those blocks to the light client contract on Calimero shard.
+<img src={require('@site/static/img/relayer_simple.jpg').default} />
 
-Similarly, the Calimero to NEAR relayer reads latest blocks on the Calimero shards and relays those blocks to the light client contract on the NEAR blockchain.
+## NEAR to Calimero Relayer
+The NEAR to Calimero relayer specifically focuses on relaying blocks from the NEAR blockchain to the designated light client contract on the Calimero shard. By actively monitoring the NEAR blockchain for the latest blocks, the relayer ensures that the most up-to-date information is relayed to the Calimero shard, enabling cross-chain communication and synchronization.
```

Comment:
> the relayer ensures that the most up-to-date information is relayed to the Calimero shard

Insert here something that says that information is relayed when needed (Calimero event occurred on the opposite side or one in the epoch)